samples/tcp-echo/src/main.go

```diff
@@ -25,6 +25,10 @@ import (
 
 // main serves as the program entry point
 func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("you should run it like: 'main starting_port,ending_port prefix' ")
```

Comment:
It can actually accept more than 2 ports so the description is a bit misleading.

Reply:
Maybe it should be port 1[,port 2]... ? I'm not sure of the best way to say a string of comma separated integers representing ports.

Reply:
`[PORTS...]` with a text description that ports are comma-delineated could be sufficient

Reply:
thanks all for the review.
Push again.
like 
```
if len(os.Args) != 3 {
		fmt.Println("you should run it like: 'main [PORTS...] prefix' , please notice PORTS are comma-delineated , like  port 1[,port 2]  ")
		os.Exit(1)
	}
```.

Would you think it is ok?